int32 grasp
Pose obj_pose
Pose2D rb_pose
---
float32 reach
Pose2D gbp
